Command line tool example

halpomeranz opened the pull request:
I really wanted a command-line tool to interface with your msi-keyboard code and thought other people might find it useful. So I'm contributing it in the examples directory.

stevelacy commented:
Thanks for adding it!

Please ensure the code follows this guide: https://github.com/felixge/node-style-guide

halpomeranz commented:
I think I cleaned the code to follow the suggested style guide. Let me know if I missed anything.
I just dropped the #! at the front of the code, since you're correct that it's not generally portable. Folks can add their own or just "node[js] msi-kbd.js"

stevelacy commented:
Thanks, merged!
